iati/fetch/__init__.py

```diff
@@ -97,3 +98,25 @@ def is_response_okay(request):
         return False
     else:
         return True
+
+
+def get_publishers(registry_api_endpoint=REGISTRY_API_LIST_PUBLISHERS):
+    """Get a list of publishers who are publishing to the IATI Registry.
+
+    Args:
+        registry_api_endpoint (str): The API endpoint URL for obtaining a list of all organisations publishing to the IATI Registry.
+
+    Raises:
+        Exception: When a HTTP 200 response code is not received.
+
+    Returns:
+        list: List of Registry publisher IDs.
+    """
```

Comment:
On reflection I think it's ok to not include a todo here. 

There is already a todo in the `is_response_okay` docstring. Also, when the function is removed all usage of the function should be replaced anyway.